src/Microsoft.AspNet.Mvc.Core/ViewContext.cs

```diff
@@ -48,11 +51,13 @@ public class ViewContext : ActionContext
         /// <param name="viewContext">The <see cref="ViewContext"/> to copy values from.</param>
         /// <param name="view">The <see cref="IView"/> being rendered.</param>
         /// <param name="viewData">The <see cref="ViewDataDictionary"/>.</param>
+        /// <param name="tempData">The <see cref="ITempDataDictionary"/>.</param>
         /// <param name="writer">The <see cref="TextWriter"/> to render output to.</param>
         public ViewContext(
             [NotNull] ViewContext viewContext,
             [NotNull] IView view,
             [NotNull] ViewDataDictionary viewData,
+            [NotNull] ITempDataDictionary tempData,
             [NotNull] TextWriter writer)
```

Comment:
This is the copy constructor to create a 'child' ViewContext, IMO we should just copy the TempData.